### ISSUE TITLE
Ability to build binaries not in root directory

### ISSUE THREAD

**Issue by cohix** [tj/gobinaries#18]
## Prerequisites

* [x] I searched to see if the issue already exists.

## Description

Not all projects have their `main.go` in the root; I would love to see some way to allow projects to specify where the entrypoint is using a `.gobinaries` file or something!

## Steps to Reproduce
```
> curl -sf https://gobinaries.com/suborbital/hivew | sh

  ==> Downloading github.com/suborbital/hivew@master
  ==> Resolved version master to v0.0.1
  ==> Downloading binary for darwin amd64

  Error downloading, got 500 response from server
```
## Slack

Join us on Slack https://chat.apex.sh/

**Comment by cohix** [comment]
Ahh I can see that passing said subpath does work.....
```
> curl -sf https://gobinaries.com/suborbital/hivew/hivew | sh

  ==> Downloading github.com/suborbital/hivew/hivew@master
  ==> Resolved version master to v0.0.1
  ==> Downloading binary for darwin amd64
  ==> Installing hivew to /usr/local/bin
Password:
  ==> Installation complete
```
That honestly might be enough, but should the onus be on the end user to figure that out?

**Comment by tj** [comment]
Currently yeah you need the full path, such as `curl -sf https://gobinaries.com/tj/staticgen/cmd/staticgen | sh`, I guess the authors would be expected to put this in the Readme. 

I don't think many projects have multiple commands but that's also an issue for the ones that do, and some kind of special config would diverge from `go get`
